[docs][mcp] guides for SSE and streamable HTTP transport

greedy52 opened the pull request:
related:
- #56588

Amplify Preview: https://steve-56588-docs.d3b94eevwi10ji.amplifyapp.com/enroll-resources/mcp-access/

greedy52 committed:
[docs][mcp] guides for SSE and streamable HTTP transport

greedy52 commented on docs/pages/enroll-resources/mcp-access/jwt.mdx:
```diff
@@ -0,0 +1,36 @@
+(!docs/pages/includes/application-access/jwt-validate.mdx!)
+
+See the example Go program used to validate Teleport's JWT tokens on our
+[GitHub](https://github.com/gravitational/teleport/blob/v(=teleport.version=)/examples/mcp-servers/).
```
examples PR in progress
- https://github.com/gravitational/teleport/pull/60165

I will drop this link if the above PR doesn't make it in time

greedy52 committed:
add a screenshot for Teleport Connect

ptgott commented on docs/pages/enroll-resources/mcp-access/getting-started.mdx:
```diff
@@ -116,35 +117,15 @@ Name              Description
-session. You can interact with it using sample questions like "can you show some
-details on this teleport demo?":
+You can interact with it using sample questions like "can you show some details
+on this teleport demo?":
```
```suggestion
on this Teleport demo?":
```

ptgott commented on docs/pages/enroll-resources/mcp-access/sse.mdx:
```diff
@@ -0,0 +1,80 @@
+server using `tsh`. Once successfully authorized, `tsh` establishes a session
+with the Application Service.
+
+The Application Service first starts a SSE connection to the remote MCP server
```
Do we pronounce each letter of "SSE", or is it pronounced something like "see"? If it's the former, we'll want to say "an SSE connection". If it's the latter, "a SSE" is fine.

ptgott reviewed (approved):
I would use the `sidebar_label` frontmatter field to make sure the MCP Servers section of the sidebar tells more of a story. Right now, it's hard for me to find a structure from the titles themselves, since there's a lot of repetition and noise in the sidebar labels:

<img width="295" height="453" alt="image" src="https://github.com/user-attachments/assets/2866617a-c5a5-4737-8aae-327dda7045a9" />

greedy52 committed:
address feedback

greedy52 commented:
> I would use the `sidebar_label` frontmatter field to make sure the MCP Servers section of the sidebar tells more of a story. Right now, it's hard for me to find a structure from the titles themselves, since there's a lot of repetition and noise in the sidebar labels:

Moved RBAC page back up and changed to `Enroll XXXX` for the main guides:

<img width="285" height="412" alt="Screenshot 2025-10-24 at 3 09 36 PM" src="https://github.com/user-attachments/assets/e50035f0-9536-4798-8b9f-bb97782dffb3" />

Now from top to bottom is the order we would like user to go through. and now it feels like the pages are divided to 3 sections: quick dive, main enroll guides, misc.

greedy52 committed:
review comments from Gabriel

ravicious reviewed (approved):
bot
